### ISSUE TITLE
LPE mode with specific px4 parameters

### ISSUE THREAD

**Issue by mandelyoni** [mathsten/OsloMet-Digital-Twin#2]
Hi Mathias,

We're a team in the Autonomous Systems lab at Tel-Aviv University, and we're trying to follow your instructions for flying the Intel Aero based on Optitrack.
We can't find most of the px4 parameters you recommended to change. Should we use a specific build of the PX4 firmware?

Thanks, Yoni

**Comment by mathsten** [comment]
Hi! Did you switch state estimators?

**Comment by mandelyoni** [comment]
Hi, yes, we did.
Can you send us a link to the firmware you used?

**Comment by mathsten** [comment]
We used firmware version 1.6.5.

**Comment by mandelyoni** [comment]
The one that came with the drone?

**Comment by mandelyoni** [comment]
The firmware that came with the drone?

Thank you

**Comment by mathsten** [comment]
No, the newest available at that time.

**Comment by mandelyoni** [comment]
And you built it with:  make aerofc-v1_default  ? 
and then did you upload the result .px4 file  to the drone according to Intel's recommendation:
cd /etc/aerofc/px4/
aerofc-update.sh nuttx-aerofc-v1-default.px4

Thank you very much for your help

**Comment by mathsten** [comment]
Yes, as I remember, er only followed the provided wiki from Intel step by step. We did experience some of the options only becoming visible after some other options were activated. But I do not remember the specifics :(

**Comment by YonimUni** [comment]
Thank you :)
